website/client/tests/unit/libs/highlightUsers.spec.js

```diff
@@ -17,6 +17,24 @@ describe('highlightUserAndEmail', () => {
     expect(result).to.contain('<span class="at-text at-highlight">@user</span>');
   });
 
+  it('highlights username sandwiched with underscores', () => {
+    const text = 'hello @_user_';
```

Comment:
This is the wrong input data. Remember that highlightUsers is being called with the __output__ of habiticaMarkdown.parse -> This means that at this point in call stack, the underscores would already have been replaced by emphasis and strong tags. The correct input at this point would be `'hello @<em>user</em>'`.
```suggestion
    const text = 'hello @<em>user</em>';
```

Reply:
Oh gosh yes that makes much more sense. I've added in your suggestions. Thank you so much for all of your help thus far!!